src/ints/bios_disk.cpp

```diff
@@ -190,16 +190,14 @@ uint8_t imageDisk::Write_Sector(uint32_t head,uint32_t cylinder,uint32_t sector,
 
 
 uint8_t imageDisk::Write_AbsoluteSector(uint32_t sectnum, void *data) {
-	uint32_t bytenum;
-
-	bytenum = sectnum * sector_size;
+	cross_off_t bytenum = static_cast<cross_off_t>(sectnum) * sector_size;
```

Comment:
Same comment as above (do we modify `bytenum` later on? ideally it can be `const auto`)